sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestBatching.java

```diff
@@ -67,6 +67,21 @@ public void testInsertSingleIteratorNoTx() {
         assertThat(b.size()).isEqualTo(2);
     }
 
+    @Test
+    public void testBindConstantNull() {
+        UsesBatching b = handle.attach(UsesBatching.class);
+        List<Integer> ids = Arrays.asList(1, 2, 3, 4, 5);
+
+        b.withConstantValue(ids, null);
+
+        assertThat(b.size()).isEqualTo(5);
```

Comment:
isn't there a `hasSize`?

Reply:
This isn't a Collection, so that doesn't apply.